tutorial/queriers.md

```diff
@@ -17,7 +17,7 @@ import (
 	abci "github.com/tendermint/tendermint/abci/types"
 )
 
-// query endpoints supported by the governance Querier
+// query endpoints supported by the nameservice Querier
```

Comment:
If you are making a change in the code in the `.md` files, please make a corresponding change in the `.go` file.